extensions/wikia/ImageReviewMercury/ImageReviewMercury.class.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+class ImageReviewMercury extends SpecialPage {
```

Comment:
If you asked me, I would say it is so that the rest of ImageReview* classes would work fine as a Special Page. Although I somewhat do not like having a class just to have a name :-)